app/pages/project/index.cjsx

```diff
@@ -97,14 +98,19 @@ ProjectPageController = React.createClass
 
     slug = ownerName + '/' + projectName
 
-    apiClient.type('projects').get({ slug, include: 'avatar,background,owners' })
+    apiClient.type('projects').get({ slug, include: 'avatar,background,organization,owners' })
       .then ([project]) =>
         @setState {project}
 
         if project?
           # Use apiClient with cached resources from include to get out of cache
           awaitBackground = apiClient.type('backgrounds').get(project.links.background.id).catch((error) => [])
 
+          if project.links.organization?
+            awaitOrganization = project.get('organization', { listed: true }).catch((error) => [])
```

Comment:
The caching layer in the client won't be used if you use the resource to get the organization. So I think right now two API requests are happening for the org, once with the include on line 101 and once here. Since you need to include this query param, then I think this line should remain and the addition to the include for the projects request should be removed.

Reply:
not sure I've implemented the most elegant solution, but appears to be working as intended, let me know what you think